src/angularPrismicio.js

```diff
@@ -291,6 +324,7 @@ angular.module('prismic.io', [])
         Configurer.init(service, config);
         service.api = api;
         service.ctx = ctx;
+        service.getAuthenticationUrl = getAuthenticationUrl;
```

Comment:
None of the properties are prefixed with get* or set*
I would suggest to rename it to authenticationUrl.